README.md

```diff
@@ -11,13 +12,15 @@ This is documentation for:
 
 ## Dev Installation instructions
 
-1. `git clone git@github.com:RedHatInsights/insights-frontend-storybook.git`
-2. `npm install`
-3. (optional) set up [insights-proxy](https://github.com/RedHatInsights/insights-proxy)
-* only required for insights-chrome's stories to run
+1: `git clone git@github.com:RedHatInsights/insights-frontend-storybook.git`
```

Comment:
Should be `1.` so it will be shown as ordered list. Now it's just string which spans over one line and looks kinda off
![storybook-snap](https://user-images.githubusercontent.com/3439771/45714822-03044c00-bb93-11e8-876f-089b3a553915.JPG)